surah/models.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+'''This module contains Surahs of Holy Quran'''
```

Comment:
The description is not descriptive at all. It seems like we are storing some data in this file.